src/cleanup-java.ts

```diff
@@ -0,0 +1,16 @@
+import * as core from '@actions/core';
+import * as gpg from './gpg';
+
+async function run() {
+  if (core.getInput('gpg-private-key', {required: false})) {
+    console.log('removing private key from keychain');
```

Comment:
There is a `core.info` method that can be used over `console.log` https://github.com/actions/toolkit/tree/master/packages/core#logging

Does the same thing effectively, but we prefer to use it.